### ISSUE TITLE
`unreachable executed` error when using MultiBody joint with WASM

### ISSUE THREAD

**Issue by KyroVibe** [dimforge/rapier.js#248]
### Description
I'm currently testing out Rapier inside of a react app using the `@dimforge/rapier3d-compat` (v^0.11.2) NPM package. I have a scene setup using impulse joints and when I swap the creation method for impulse joints out for multibody joint, I get an `unreachable executed` error some where in rapiers WASM code:
![a](https://github.com/dimforge/rapier.js/assets/20159878/1feb8202-0a45-4f46-9abe-e35bb6fcc143)
The line in question is a return statement with the new joint:
![b](https://github.com/dimforge/rapier.js/assets/20159878/e5a755fa-fc6d-4541-8975-e173970cfe40)
Any help would be greatly appreciated!

**Comment by MaxBittker** [comment]
did you figure this out?

**Comment by KyroVibe** [comment]
@MaxBittker Nope. I isolated the issue to be something involving the borrow-checker, but was unable to see what I was doing wrong. I made my own WASM library for Rapier and that fixed it, but still ran into some other issues with static bodies and their behavior in MultiBody joints (which I believe is something wrong with the core library). My team ended up ditching Rapier altogether for JoltPhysicsJS.

**Comment by amraei** [comment]
> [@MaxBittker](https://github.com/MaxBittker) Nope. I isolated the issue to be something involving the borrow-checker, but was unable to see what I was doing wrong. I made my own WASM library for Rapier and that fixed it, but still ran into some other issues with static bodies and their behavior in MultiBody joints (which I believe is something wrong with the core library). My team ended up ditching Rapier altogether for JoltPhysicsJS.

I'm eager to know if you're still ok with JoltPhysicsJS and do you still recommend it?

**Comment by HunterBarclay** [comment]
@amraei Definitely recommend it. Rapier has a lot of potential, but for my teams application, constraint based physics is a must, and Multibodies in rapier (and joints in general) aren't nearly as well flushed out as they are in Jolt.
